src/app/explore/explore.component.html

```diff
@@ -17,12 +17,19 @@
     </biggive-totalizer>
   }
 
-  @if (metaCampaign && metaCampaign.useDon1120Banner) {
+  @if (metaCampaign && !!metaCampaign.bannerLayout) {
     <app-meta-campaign-banner
       [logo]="fund?.logoUri ? { url: fund?.logoUri!, alt: fund?.name } : undefined"
       [mainTitle]="title"
       [teaser]="fund && fund.description ? fund.description : metaCampaign.summary"
       [mainImageUrl]="metaCampaign.bannerUri"
+      [backgroundColour]="metaCampaign.bannerLayout.backgroundColour"
+      [focalPoint]="{
+        x: metaCampaign.bannerLayout.focalArea.topLeftXpos,
+        y: metaCampaign.bannerLayout.focalArea.topLeftYpos,
```

Comment:
for now just taking the top left position, using the bottom right as well is for the future but I'm not sure if we'll actually need it - it depends whether the size of the box will actually vary enough between metacampaigns to matter.

Reply:
might have been better to pass the entire bannerLayout object into the new component as a single input. I don't think that should be a blocker though.